packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1063,26 +1063,42 @@ oie.enroll.okta_verify.setup.skipAuth.selectAccount = In the app, select your ac
 oie.enroll.okta_verify.setup.skipAuth.addAccount = Choose <$1>Add Account to Another Device.</$1>
 oie.enroll.okta_verify.setup.skipAuth.followInstruction = Follow the rest of the instructions shown in Okta Verify.
 oie.enroll.okta_verify.setup.skipAuth.canBeClosed = This screen can be closed at any time.
+oie.enroll.okta_verify.setup.customUri.makeSureHaveOVToContinue = To continue, make sure you have the Okta Verify app.
+oie.enroll.okta_verify.setup.customUri.setup = Once Okta Verify is installed, return to this page and tap <$1>Set up Okta Verify</$1>.
+oie.enroll.okta_verify.setup.customUri.prompt = Click <$1>"Open Okta Verify"</$1> on the browser prompt to set up Okta Verify.
+oie.enroll.okta_verify.setup.customUri.noPrompt = <$1>Didn't see the prompt?</$1>
+oie.enroll.okta_verify.setup.customUri.makeSureHaveOV = Make sure you have Okta Verify installed.
+oie.enroll.okta_verify.setup.customUri.orMobile = Or <$1>set up Okta Verify on another mobile device</$1>
 oie.enroll.okta_verify.select.channel.description = Which option do you want to try?
 oie.enroll.okta_verify.select.channel.qrcode.label = Scan a QR code
+oie.enroll.okta_verify.select.channel.subtitle = How would you like to set up Okta Verify?
 oie.enroll.okta_verify.select.channel.email.label = Email me a setup link
 oie.enroll.okta_verify.select.channel.sms.label = Text me a setup link
+oie.enroll.okta_verify.select.channel.ovOnThisDevice = Or <$1>set up Okta Verify on this device</$1>
 oie.enroll.okta_verify.select.channel.title = More options
+oie.enroll.okta_verify.select.channel.title.updated = Set up Okta Verify on another mobile device
 oie.enroll.okta_verify.channel.email.label = Email
 oie.enroll.okta_verify.enroll.channel.email.title = Set up Okta Verify via email link
+oie.enroll.okta_verify.enroll.channel.email.subtitle = Make sure you can access the email on the receiving device
 oie.enroll.okta_verify.enroll.channel.sms.title = Set up Okta Verify via SMS
 oie.enroll.okta_verify.channel.email.description = Make sure you can access the email on your mobile device.
 oie.enroll.okta_verify.channel.sms.description = Make sure you can access the text on your mobile device.
+oie.enroll.okta_verify.channel.sms.description.updated = Make sure you can access the text on the receiving device
+oie.enroll.okta_verify.qrcode.title = Set up Okta Verify via QR code
 oie.enroll.okta_verify.qrcode.step1 = On your mobile device, download the Okta Verify app from the App Store (iPhone and iPad) or Google Play (Android devices).
+oie.enroll.okta_verify.qrcode.step1.updated = On your other device, download the Okta Verify app from the App Store� (iPhone� and iPad�) or on Google Play (Android? devices).
```

Comment:
@glenfannin-okta , @lesterchoi-okta , @shawyu-okta please confirm we can use unicode here for trademark symbols

Reply:
Looks like we have one that currently exists in the login.properties file so this should be safe.